ietf/meeting/tests_views.py

```diff
@@ -1221,7 +1221,55 @@ def test_edit_schedule(self):
         self.client.login(username="secretary", password="secretary+password")
         r = self.client.get(urlreverse("ietf.meeting.views.edit_schedule", kwargs=dict(num=meeting.number)))
         self.assertContains(r, "load_assignments")
- 
+
+    def test_official_record_schedule_is_read_only(self):
+        def _set_date_offset_and_retrieve_page(meeting, days_offset, client):
+            meeting.date = datetime.date.today() + datetime.timedelta(days=days_offset)
+            meeting.save()
+            client.login(username="secretary", password="secretary+password")
+            url = urlreverse("ietf.meeting.views.edit_meeting_schedule", kwargs=dict(num=meeting.number)) 
+            r = client.get(url)
+            q = PyQuery(r.content)
+            return(r, q)
+
+        # Setup
+        ####################################################################################
+
+        # Basic test data
+        meeting = make_meeting_test_data()
+
+        # Set the secretary as the owner of the schedule
+        schedule = meeting.schedule
+        schedule.owner = Person.objects.get(user__username="secretary")
+        schedule.save()
+
+        # Tests
+        ####################################################################################
+
+        # 1) Check that we get told the page is not editable
+        #######################################################
+        r, q = _set_date_offset_and_retrieve_page(meeting,
+                                                  0 - 2 - meeting.days, # Meeting ended 2 days ago
+                                                  self.client)
+        self.assertTrue(q("em:contains(\"You can't edit this schedule\")"))
```

Comment:
You might consider using single quotes for the outer quotes to avoid needing to escape the inner quotes.

Reply:
Yes, but after I use single quotes for the outer quote to avoid escaping the double quotes in the inner quote the inner single quote needs to be double quoted so that it doesn't end the outer quotes.

Ah, but I just learned that one can triple-double-quote a string with both quotes and double quotes without having to escape the quotes or double quote the single quotes.  I'll do that!

Reply:
"EGAD"